cmd/oklog/forward.go

```diff
@@ -21,8 +21,10 @@ import (
 func runForward(args []string) error {
 	flagset := flag.NewFlagSet("forward", flag.ExitOnError)
 	var (
-		apiAddr  = flagset.String("api", "", "listen address for forward API (and metrics)")
-		prefixes = stringslice{}
+		apiAddr     = flagset.String("api", "", "listen address for forward API (and metrics)")
+		prefixes    = stringslice{}
+		ringBuf     = flagset.Bool("buf", false, "use a ring buffer to drop old messages when e.g. the ingestion engine fails")
+		ringBufSize = flagset.Int("bufsize", 1024, "ring buffer size (messages)")
```

Comment:
Yup, sure. Sounds like you'd like to keep options open for other alternatives in future.